### ISSUE TITLE
How to run Rancher agent and host on the same server

### ISSUE THREAD

**Issue by pensiero** [rancher/rancher#13829]
**Rancher versions:**
rancher/server or rancher/rancher: `2.0.2`

I tried running both the agent and host on the same server, Rancher works but when I configure an Nginx Ingress Controller, with custom host mapping, and I try to reach the specified host, it display the Rancher dashboard.

**Comment by OBenned** [comment]
Are u running docker run command or something else?

Maybe try and run it like so:

`sudo docker run -d --restart=unless-stopped -p 8080:80 -p 8443:443 rancher/rancher`

**Comment by pensiero** [comment]
I did it with the following command: 
```
sudo docker run -d --restart=unless-stopped -p 8080:80 -p 8443:443 rancher/rancher
```

Now I managed to create a Ingress Controller and get it working. Many of the troubles were caused by the editing process of an existing Ingress or Workload. If you create from scratch, in a proper way, it works fine, but if you create and then you edit it, sometimes it breaks.

I'm sorry not to be more specific about that, I tried so many different ways that I can't remember the exact process in order to simulate the problems. I'll poke around a bit in order to reproduce it.
